requirements.txt

```diff
@@ -38,6 +38,8 @@ https://github.com/PanQiWei/AutoGPTQ/releases/download/v0.4.2/auto_gptq-0.4.2+cu
 https://github.com/PanQiWei/AutoGPTQ/releases/download/v0.4.2/auto_gptq-0.4.2+cu117-cp310-cp310-linux_x86_64.whl; platform_system == "Linux" and platform_machine == "x86_64"
 https://github.com/jllllll/exllama/releases/download/0.0.17/exllama-0.0.17+cu117-cp310-cp310-win_amd64.whl; platform_system == "Windows"
 https://github.com/jllllll/exllama/releases/download/0.0.17/exllama-0.0.17+cu117-cp310-cp310-linux_x86_64.whl; platform_system == "Linux" and platform_machine == "x86_64"
+https://github.com/turboderp/exllamav2/releases/download/v0.0.4/exllamav2-0.0.4+cu117-cp310-cp310-win_amd64.whl; platform_system == "Windows"
```

Comment:
Out of curiosity, why is a `and platform_machine == "x86_64"` not needed here? Also, can this line and line 11 apply simultaneously?

Reply:
Glad you caught that. I forgot that Windows uses `amd64` instead of `x86_64` for some reason.

This line will need `and platform_machine == "amd64"` and line 11 will need `and platform_machine != "amd64"`
This will also need to be applied to `requirements_noavx2.txt`.